docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+    - SDK Team provides ongoing consulting
+    - Regular syncs on development and organisational topics
+    - Successful PoC
+    - Iterative development process with SDK team contributing to PR reviews
+    - Releasing the contribution as `Beta`
+    - Iterative development. Stabilizing the API.
+    - Release the contribution as GA aka `General Availability`
+    - Keep functional scope up to date with customer requirements
+    - Maintanance and support of the contribution by **Contribution Team** or their successors
+
+## Responsibilities
+
+### SDK Team
+
+#### On-boarding
+
+  - Providing repository structure
+  - ESlint rules (so that compliant with SDK)
+  - Inherit our pipeline (Github actions)
+  - Testing framework
+  - NPM audit (vulnerability checks)
+  - License check tool
+
+
+### What's on them?
+
+#### Delivery plan
+  - What features
+  - Oder of feature delivery (so that we can anticipate PR sizes... else?)
+  - Basic architecture - some diagram and involved services
+  - Modules structure (CC, re-use module artifacts)
+
+
+- Setting up OS process?
+  - Coding and testing the feature?
+  - Aligning on release strategy?
+
+## Coding convention
+
+### Contribution programming language
+
+Contribution should happen in Typescript and endorse type-safety whenever possible.
+
+### API alignment
+
+To maintain a consistent API strategy, contribution and SDK teams align on the API. Final approval is on the SDK team.
+
+### Repository
+
+    - Repository is created on Github under SAP Organization by contribution team
+    - SDK and contribution team co-own the repository and have full access rights to it
+    - SDK team provides repository structure if needed. It's preferable in most cased.
+    - SDK team can audit repository settings and require reasonable updates to configuration
+
+### Introducing dependencies
+
+    - Dependency audit should happen as a part of PR review
+    - SDK provides a guide on dependency choices to the contributing team
+
+### Coding convention
+
+    - Contribution team should use coding convention of the SDK unless it's explicitly approved to be otherwise
+    - Most of coding convention rules should be enforced by automated code checkers and linters like ESLint and Prettier
+    - Enforcement happens via continuous integration pipeline
+    - SDK team provides guidelines on coding conventions that can't be automatically enforced
+    - SDK team shares documentation on standard patterns chosen by SDK for ... (what patterns do we use?)
+
+### Pull requests
+
+    - Pull request should contain some logical and complet pile of work
+    - Pull request should be **small**  for easy review
+    - Preferably Pull request should contain tangible incremental value
+    - Pull request are mandatorily reviewed by at least one of SDK team members
+
+### Branching strategy
+
+  - `Main` branch has to be always **release ready**. In other words: up to date, tested, and documented
```

Comment:
Here I would explain a bit:
- Main branch used for release (Major, Minor and Patches) So no hotfix branches etc.
- Fixes and Feature are included by feature branches which are merged
- Direct pushes to master should be the absolute exception.